README.md

```diff
@@ -17,13 +17,13 @@ By leveraging [cookiecutter](https://www.cookiecutter.io) you will be asked a sm
 
 ## Requirements
 
-```bash
+```console
```

Comment:
> @snapp Why are we doing this?

I updated the commit message (920b93d) with my reasoning:

The decision to switch from `bash` to `console` was two-fold:

* use of a generic term rather than a linux specific shell
* aesthetic preference